hugo/layouts/shortcodes/js.html

```diff
@@ -54,7 +54,7 @@
   };
 
   fillCodeRefs = (xs, parent) => {
-    if (xs.length == 0 && parent.innerText.length != 0) {
+    if (xs.length == 0 && parent.textContent.length != 0) {
```

Comment:
Nit: This will throw if `xs` or `parent` is `undefined` and does not have the length `property`, since I don't have the full context of the usage here I'm going to assume its okay.

Reply:
I'm not familiar with the code either, but the only usage is this a few lines below
```js
let codeBlocks = document.getElementsByTagName("code");
for (let codeBlock of codeBlocks) {
    fillCodeRefs(codeBlock.children, codeBlock);
}
```
Seems unlikely that `parent` is undefined and all DOM Nodes have `textContent`